lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -345,6 +348,7 @@ export class WindowUtils {
         // Office.js sets history.replaceState to null
```

Comment:
We could make this function generic and take in a window object, so that it could be used for popups and redirects

Reply:
Done!